src/core/getExports.js

```diff
@@ -236,18 +236,19 @@ export default class ExportMap {
     if (this.reexports.has(name)) return true
 
     // default exports must be explicitly re-exported (#328)
+    let foundInnerMapName = false
     if (name !== 'default') {
-      for (let dep of this.dependencies.values()) {
-        let innerMap = dep()
+      this.dependencies.forEach((dep) => {
```

Comment:
this would ideally be a `some` so that it short-circuits. not sure how to achieve with a `Set`...